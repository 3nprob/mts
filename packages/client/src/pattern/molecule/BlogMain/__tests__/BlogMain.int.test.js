import { regular } from './BlogMain.int.story'

describe('<BlogMain />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
