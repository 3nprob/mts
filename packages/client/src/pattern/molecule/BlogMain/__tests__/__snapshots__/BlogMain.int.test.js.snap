// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogMain /> Snaphot must match defaultRender 1`] = `
<main
  className=""
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
      "marginTop": "55px",
    }
  }
>
  <div
    className="container1"
  >
    <section
      itemScope={true}
      itemType="http://schema.org/Blog"
    >
      <h2>
        <a
          href="/categories/recent"
          itemProp="url"
        >
          <span>
            Recent articles
            <svg
              height="16px"
              viewBox="0 0 185.343 185.343"
              width="16px"
            >
              <path
                d="M51.707 185.343a10.692 10.692 0 0 1-7.593-3.149 10.724 10.724 0 0 1 0-15.175l74.352-74.347L44.114 18.32c-4.194-4.194-4.194-10.987 0-15.175 4.194-4.194 10.987-4.194 15.18 0l81.934 81.934c4.194 4.194 4.194 10.987 0 15.175l-81.934 81.939a10.678 10.678 0 0 1-7.587 3.15z"
                fill="#823eb7"
              />
            </svg>
          </span>
        </a>
      </h2>
      <article
        className="container1BlogItem container1BlogItemFeatured"
      >
        <div
          className="container1BlogItemImage container1BlogItemImageFeatured"
        >
          <div
            style={
              Object {
                "paddingBottom": "64.94140625%",
                "width": "100%",
              }
            }
          />
          <picture>
            <img
              alt="blog-1-0"
              src={Object {}}
            />
          </picture>
        </div>
        <div
          className="container1BlogItemContent container1BlogItemContentFeatured"
        >
          <header>
            <h2
              itemProp="name headline"
            >
              <a
                aria-label="Go To Blog Post"
                href="/react-libraries"
                itemProp="url"
              >
                Lorem ipsum
              </a>
            </h2>
          </header>
          <p
            className=""
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
          </p>
        </div>
      </article>
      <div
        className="container1BlogList"
      >
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-1-1"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/conditional-rendering-react"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-1-2"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/react-event-handler"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-1-3"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/javascript-import-export"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
      </div>
    </section>
    <section
      itemScope={true}
      itemType="http://schema.org/Blog"
    >
      <h2>
        <a
          href="/categories/starter"
          itemProp="url"
        >
          <span>
            Getting Started Tutorials
            <svg
              height="16px"
              viewBox="0 0 185.343 185.343"
              width="16px"
            >
              <path
                d="M51.707 185.343a10.692 10.692 0 0 1-7.593-3.149 10.724 10.724 0 0 1 0-15.175l74.352-74.347L44.114 18.32c-4.194-4.194-4.194-10.987 0-15.175 4.194-4.194 10.987-4.194 15.18 0l81.934 81.934c4.194 4.194 4.194 10.987 0 15.175l-81.934 81.939a10.678 10.678 0 0 1-7.587 3.15z"
                fill="#823eb7"
              />
            </svg>
          </span>
        </a>
      </h2>
      <article
        className="container1BlogItem container1BlogItemFeatured"
      >
        <div
          className="container1BlogItemImage container1BlogItemImageFeatured"
        >
          <div
            style={
              Object {
                "paddingBottom": "64.94140625%",
                "width": "100%",
              }
            }
          />
          <picture>
            <img
              alt="blog-2-0"
              src={Object {}}
            />
          </picture>
        </div>
        <div
          className="container1BlogItemContent container1BlogItemContentFeatured"
        >
          <header>
            <h2
              itemProp="name headline"
            >
              <a
                aria-label="Go To Blog Post"
                href="/node-js-express-tutorial"
                itemProp="url"
              >
                Lorem ipsum
              </a>
            </h2>
          </header>
          <p
            className=""
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
          </p>
        </div>
      </article>
      <div
        className="container1BlogList"
      >
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-2-1"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/learn-react-js"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-2-2"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/react-redux-tutorial"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
        <article
          className="container1BlogListContainer1"
        >
          <div
            className="container1BlogListContainer1Image"
          >
            <div
              style={
                Object {
                  "paddingBottom": "6.69921875%",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="blog-2-3"
                src={Object {}}
              />
            </picture>
          </div>
          <div
            className="container1BlogListContainer1Content"
          >
            <header>
              <h2
                itemProp="name headline"
              >
                <a
                  aria-label="Go To Blog Post"
                  href="/complete-firebase-authentication-react-tutorial"
                  itemProp="url"
                >
                  Lorem ipsum
                </a>
              </h2>
            </header>
            <p
              className=""
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
            </p>
          </div>
        </article>
      </div>
    </section>
  </div>
</main>
`;
